market_maker_keeper/erisx_market_maker_keeper.py

```diff
@@ -385,23 +394,8 @@ def place_order_function(new_order_to_be_placed):
             else:
                 logging.info(f"New {side} Order below size minimum of {min_amount}. Order of amount {amount} ignored.")
 
-    def sync_order_book(self) -> List[Order]:
-        """
-           Check the list of orders for cancellations or fills.
-           If an order has been partially filled, the order amount will be updated.
-
-           If an application message has been received from ErisX,
-           update the Keeper orderbook accordingly.
-        """
-        existing_orders = self.orders
-        self.orders = self.erisx_api.sync_orders(existing_orders)
-
     def synchronize_orders(self):
         bands = Bands.read(self.bands_config, self.spread_feed, self.control_feed, self.history)
-
-        # check for any cancelled or filled orders
-        self.sync_order_book()
```

Comment:
This needn't be done here because `ErisXOrderBookManager` has a thread which maintains the order book.  In it, `get_orders_function` calls the `ErisXMarketMakerKeeper.get_orders` (here), which runs `erisx_api.sync_orders` (pyexchange).

Reply:
Yes, that mismatch had led to the orderbook being out of sync so I removed this code and moved it all to `get_orders()` and retested.